packages-content-model/roosterjs-content-model-dom/lib/domToModel/processors/entityProcessor.ts

```diff
@@ -23,22 +23,7 @@ export const entityProcessor: ElementProcessor<HTMLElement> = (group, element, c
         context,
         { segment: isBlockEntity ? 'empty' : undefined, paragraph: 'empty' },
         () => {
-            const wrapperToUse = context.allowCacheElement
-                ? element
-                : (element.cloneNode(true /* deep */) as HTMLElement);
-
-            if (!context.allowCacheElement) {
-                wrapperToUse.style.backgroundColor = element.style.backgroundColor || 'inherit';
-                wrapperToUse.style.color = element.style.color || 'inherit';
```

Comment:
I think maybe this is what might be causing the above issue.

Reply:
You are right, I also just debugged to here.